src/mnode/ticket-processor.cpp

```diff
@@ -750,7 +750,7 @@ bool CPastelTicketProcessor::CreateP2FMSTransactionWithExtra(const CDataStream&
     int fake_key_size = 33;
     size_t non_padded_size = input_bytes.size();
     size_t padding_size = fake_key_size - (non_padded_size % fake_key_size);
-    if (padding_size != 0) {
+    if (padding_size != 0) { //-V547 false warning
```

Comment:
In case that (non_padded_size % fake_key_size) == 0. A redundant chunk with 33 bytes will be insert. Should change the implementation.

Reply:
so do you want to change this as if(non_padded_size % fake_key_size != 0)?

Reply:
Yes I believe that is the best fix.